packit_service/constants.py

```diff
@@ -262,3 +262,14 @@ def from_number(number: int):
 # shared sandcastle dir
 SANDCASTLE_DG_REPO_DIR = "dist-git"
 SANDCASTLE_LOCAL_PROJECT_DIR = "local-project"
+
+ERROR_COMMENT_MESSAGE_DETAILS = (
+    "commit_sha",
+    "copr_build_packit_info",
+    "srpm_build_packit_info",
+    "copr_build_logs",
```

Comment:
With Copr, we differentiate logs URL and web URL. This would be nice to cover as well.

Reply:
If I go with the Laura's suggestion, then I can think to have 3 links:

`packit_dashboard_url`
`logs_url`
`external_dashboard_url` -> the copr web url can be linked here? And maybe also the koji dashboard build detail, that I think we have?

Reply:
Yes, something like that would be nice.